src/test/scala/mesosphere/marathon/api/akkahttp/v2/PodsControllerTest.scala

```diff
@@ -55,7 +59,12 @@ class PodsControllerTest extends UnitTest with ScalatestRouteTest with RouteBeha
       val request = Post(Uri./.withQuery(Query("force" -> "false")))
         .withEntity(entity)
         .withHeaders(`Remote-Address`(RemoteAddress(InetAddress.getByName("192.168.3.12"))))
+
+      val podDefinition = PodDefinition(id = PathId("mypod"))
+      f.podManager.find(any).returns(Some(podDefinition))
```

Comment:
How about this? `f.podManager.find(any)` -> `f.podManager.find(eq(PathId("mypod")))`

Reply:
Same as in #5754. We do not want to test whether `find` was invoked but if the request is rejection due to unauthorized requests.